sqlite3/lib/src/impl/database.dart

```diff
@@ -294,6 +294,29 @@ class DatabaseImpl implements Database {
     return allocateBytes(functionNameBytes, additionalLength: 1);
   }
 
+  @override
+  void createCollation({
+    required String collateName,
+    required CollatingFunction function,
+  }) {
+    final storedFunction = functionStore.registerCollating(function);
+    final namePtr = _functionName(collateName);
+
+    final result = _bindings.sqlite3_create_collation_v2(
+      _handle,
+      namePtr.cast(), // zFunctionName
+      _eTextRep(false, false),
```

Comment:
This is a bit confusing since ORing the flags for determinism/direct only doesn't appear to be supported for collations. Let's just use `SqlTextEncoding.SQLITE_UTF8` here:

```suggestion
      SqlTextEncoding.SQLITE_UTF8,
```

Reply:
I'm ok with that.

For informations, the possible values are : 
- SQLITE_UTF8
- SQLITE_UTF16LE
- SQLITE_UTF16BE
- SQLITE_UTF16
- SQLITE_UTF16_ALIGNED